connect/json/src/test/java/org/apache/kafka/connect/json/JsonConverterTest.java

```diff
@@ -607,6 +611,19 @@ public void testCacheSchemaToJsonConversion() {
         assertEquals(2, cache.size());
     }
 
+    @Test
+    public void testJsonSchemaCacheSizeFromConfigFile() throws URISyntaxException, IOException {
+        URL url = getClass().getResource("/connect-test.properties");
```

Comment:
+1.. yah i get that.. but wanted to simulate exactly what happens when we configure this from a file. I'll change it if you prefer.